packages/manager/modules/pci/src/projects/project/storages/databases/database/users/modify-password/translations/Messages_fr_FR.json

```
@@ -0,0 +1,9 @@
+{
+  "pci_databases_users_modify_password_title": "Modifier le mot de passe",
+  "pci_databases_users_modify_password_confirm": "Confirmer",
+  "pci_databases_users_modify_password_cancel": "Annuler",
+  "pci_databases_users_modify_password_label": "Le mot de passe",
+  "pci_databases_users_modify_password_help_text": "Doit contenir au moins un chiffre, au moins une majuscule, au moins une minuscule. Et être constitué de 8 caractères ou plus.",
```

Because the both translation keys are to consider as a single phrase. The second translation must start with a lower char. 
```suggestion
  "pci_databases_users_modify_password_label": "Le mot de passe",
  "pci_databases_users_modify_password_help_text": " doit contenir au moins un chiffre, au moins une majuscule, au moins une minuscule. Et être constitué de 8 caractères ou plus.",
```

---

This is a separate sentence that comes below the UI field as a helper text.